launcher/src/test/java/org/apache/spark/launcher/CommandBuilderUtilsSuite.java

```diff
@@ -99,6 +99,11 @@ public void testJavaMajorVersion() {
     assertEquals(10, javaMajorVersion("10"));
   }
 
+  @Test
+  public void testGetJarsDir() {
+    assertEquals("/spark/home/jars", getJarsDir("/spark/home"));
```

Comment:
Hi, @marblejenka .
This seems to add a half of the test coverage.
Since this PR's key contribution is to introduce `ENV_SPARK_JARS_DIR`, shall we have a test coverage for `ENV_SPARK_JARS_DIR`, too?